Lib/test/test_sys.py

```diff
@@ -695,10 +692,8 @@ def test_43581(self):
         self.assertEqual(sys.__stdout__.encoding, sys.__stderr__.encoding)
 
     def test_intern(self):
-        global INTERN_NUMRUNS
-        INTERN_NUMRUNS += 1
         self.assertRaises(TypeError, sys.intern)
-        s = "never interned before" + str(INTERN_NUMRUNS)
+        s = "never interned before" + str(random.randrange(0, 10**18))
```

Comment:
Oh, for me 9 digits was already way enough. 18 digits is just too much 😁 You can just pass the max value:

```suggestion
        s = "never interned before" + str(random.randrange(10**9))
```